scanner/src/test/kotlin/ScannerTest.kt

```diff
@@ -158,9 +158,8 @@ class ScannerTest : WordSpec({
                 )
             )
 
-            verify(exactly = 1) {
-                scannerWrapper.scanPackage(pkgWithArtifact, any(), any())
-                scannerWrapper.scanPackage(pkgWithVcs, any(), any())
+            verify(exactly = 2) {
+                scannerWrapper.scanPackage(any(), any())
```

Comment:
This test now partly loses it's meaning, I think it would be good to replace the `any()` for `context` with an actual context to ensure that `context.coveredPackages` is correct (same below).